.pre-commit-config.yaml

```diff
@@ -20,6 +20,13 @@ repos:
     rev: 5.9.3
     hooks:
       - id: isort
+  - repo: local
+    hooks:
+      - id: pylint
+        name: pylint
+        entry: pylint
+        language: system
+        files: 'gitlab/'
```

Comment:
Thanks! I'll try to run this locally but does it mean it'll always run on all source files?

Reply:
Not sure I completely understand but it will run on all files located under the `gitlab/' directory. Meaning it won't check the `tests/` directory or anything else not inside the `gitlab/` directory.

Reply:
Sorry, just thought we'd have the same issue we had before where it would override what files get analyzed and ignore the linter's config. Seems to work ok!

However this is still a bit of headache now 🤦 on a clean system, it expects `pylint` installed which people might not have as we use tox for managing dependencies (see below), plus versions wouldn't match potentially. I see some people use the tox venv for this sometimes as a solution: https://github.com/pre-commit/pre-commit/issues/1309. Not sure I like that though, WDYT @JohnVillalovos should we just still add it or find a solution in a follow-up?

```
flake8...................................................................Passed
isort....................................................................Passed
pylint...................................................................Failed
- hook id: pylint
- exit code: 1

Executable `pylint` not found
```

Reply:
Fixed it. So it is working now.